tsconfig.json

```diff
@@ -10,8 +10,7 @@
     "allowSyntheticDefaultImports": true,
     "lib": [
       "DOM",
-      "ES2021",
-      "ES2024.Promise"
```

Comment:
Why are you removing this?

Reply:
I think it is by mistake will restore it!